analysis_directory/parameters_structured.tsv

```diff
@@ -204,7 +205,7 @@ pandaseqThreads 20
 #
 #=     pandaseqMinlen:
 #=        title: pandaseq minimum length
-#=        description: Sets the minimum length for a sequence, after primers are removed. By default, all sequences are kept. With this option, sequences shorter than desired can be discarded. In case you need to use this parameter, be sure you leave a tab after 'minlen' and set it like this: '-l 80'. If you do not want to use this parameter, make sure there is nothing after 'minlen'.
+#=        description: "Sets the minimum length for a sequence, after primers are removed. By default, all sequences are kept. With this option, sequences shorter than desired can be discarded. In case you need to use this parameter, be sure you leave a tab after 'minlen' and set it like this: '-l 80'. If you do not want to use this parameter, make sure there is nothing after 'minlen'."
```

Comment:
Could you replace this description with the following? 

"Sets the minimum length for a sequence, after primers are removed. Sequences shorter than desired will be discarded. By default, all sequences are kept. To do so, you may leave this parameter empty."

Reply:
yes, sure, done

but have to say: this is quite unrelated to what I was trying to achieve here and now

* I am only focusing on syntactic yaml parsing of the paramaters file so I can go back to processing it with the external yaml4parms, not on correct semantic descriptions
* my feeling is that you can safely apply these extra descriptions in your own branch & merge at any time convenient -- or else merge the PR first and then add those changes ?) 


I already hooked up a test inside the yaml4parms to the latest paramaters-structured.tsv inside pema, so I automaticallt stay in sync with your updates .

At some time (after actual release of yaml4parms I suppose the reverse hookup will make sense to (some CI/CD action in this pema repo could use the yaml4parms to check)